avro/build.gradle.kts

```diff
@@ -0,0 +1,48 @@
+description = "A library for error handling in Kafka Streams."
+
+plugins {
+    `java-library`
+    id("com.github.davidmc24.gradle.plugin.avro") version "1.2.1"
+}
+
+// add .avsc files to jar allowing us to use them in other projects as a schema dependency
+sourceSets {
+    main {
+        resources {
+            srcDirs("src/main/avro")
+        }
+    }
+}
+
+dependencies {
+    val kafkaVersion: String by project
+    api(group = "org.apache.kafka", name = "kafka-streams", version = kafkaVersion)
+    val avroVersion: String by project
+    api(group = "org.apache.avro", name = "avro", version = avroVersion)
+    implementation(project(":core"))
```

Comment:
should be implementation

Reply:
guess you mean L21?

Reply:
Ah^^ Line was correct but I messed up: Should be api